tests/migration/reboot_to_upgrade.pm

```diff
@@ -22,10 +22,15 @@ use utils;
 use Utils::Backends 'is_pvm';
 use power_action_utils 'power_action';
 use bootloader_setup 'stop_grub_timeout';
+use version_utils;
+use x11utils 'turn_off_gnome_show_banner';
 
 sub run {
     my ($self) = @_;
 
+    # We need to close gnome notification banner before migration.
```

Comment:
You'd better to include the updated code in the condition to ensure won't run on other arches.

Reply:
changed.